src/api/ledger/parse/ledger.js

```diff
@@ -0,0 +1,50 @@
+/* @flow */
+'use strict';
+const _ = require('lodash');
+const removeUndefined = require('./utils').removeUndefined;
+const parseTransaction = require('./transaction');
+
+function parseTransactions(transactions) {
+  if (!transactions || transactions.length === 0) {
+    return {};
+  }
+  if (typeof transactions[0] === 'string') {
+    return {transactionHashes: transactions};
+  }
+  return {
+    transactions: _.map(transactions, parseTransaction),
```

Comment:
Seems weird that the new API code tends to use lodash for array iteration methods strictly... in this parseTransactions() function we could use _.isEmpty() for the first conditional and _.isString() for the second...